snakeviz/cli.py

```diff
@@ -103,14 +105,20 @@ def main(argv=sys.argv[1:]):
         print('No available port found.')
         return 1
 
+    url = "http://{0}:{1}/snakeviz/{2}".format(hostname, port, filename)
     print(('snakeviz web server started on %s:%d; enter Ctrl-C to exit' %
            (hostname, port)))
+    print(unquote_plus(url))
```

Comment:
Thanks for testing that. I think I'd still prefer for the printed URL to match the URL passed to the browser, unless there's a reason not to do that?